### ISSUE TITLE
Okta react authentication sample cors error

### ISSUE THREAD

**Issue by kuttim** [okta/samples-js-react#74]
Hi, I've followed your [guide](https://developer.okta.com/blog/2018/02/06/build-user-registration-with-node-react-and-okta) I've enabled CORS on my org account, here's a preview: 
![image](https://user-images.githubusercontent.com/41844956/61128151-d753a900-a4b9-11e9-9a36-b57a87e38221.png)
However I'm getting an error when registrering an account: ```Access to XMLHttpRequest at 'https://xx-dev-753152.okta.com/api/v1/authn' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.```
any ideas?

**Comment by swiftone** [comment]
Hi @kuttim, thanks for the report.  

What status code does this request get?  If you're seeing an error result the CORS header may not be passed.  If you're seeing a 401/403, you should address what is wrong in the request first, the CORS message is a distraction.

If the status code is good (200), try clearing your local storage cache and let us know if that clears it  up or not.

**Comment by swiftone** [comment]
Closing due to lack of response.  If this issue remains a problem, please reopen the issue with more details or open a new issue.
